Firestore/core/test/unit/core/query_test.cc

```diff
@@ -48,7 +49,6 @@ using testutil::CollectionGroupQuery;
 using testutil::DbId;
 using testutil::Doc;
 using testutil::Field;
-using testutil::Filter;
```

Comment:
why did you remove this?

Reply:
Since we introduce `std::vector<Filter>` to this file. Here Filter refers to `core::Filter` (class name) instead of `testutil::Filter` (function name). With `using testutil::Filter;`, the compiler will not work.